sample/ConsoleDemo/Program.cs

```diff
@@ -11,7 +12,10 @@ public static void Main()
         {
             Log.Logger = new LoggerConfiguration()
                 .MinimumLevel.Verbose()
-                .WriteTo.Console(theme: AnsiConsoleTheme.Code)
+                .WriteTo.Console(
+                    theme: AnsiConsoleTheme.Code,
+                    outputTemplate: "[{Timestamp:HH:mm:ss} {Level:u3}] {Message}{NewLine}{Properties:lj}{NewLine}{Exception}",
+                    formatProvider: new CultureInfo("fr-FR"))
```

Comment:
I think this makes a good smoke test, but probably detracts from the usefulness of the example as a very quick/basic introduction to using the sink. Thoughts?

Reply:
You're right, I've used it mostly to validate the proper behaviour in a real app. I'll rollback to initial version. 

Thanks !